csrc/ir/internal_nodes.h

```diff
@@ -2860,6 +2860,149 @@ class ArgsortOp : public Expr {
   }
 };
 
+//! NOTE -- [ Grouped Matrix Multiplication semantics ]
+//!
+//! This operation performs a grouped matrix multiplication.
+//!
+//! Parameters:
+//! - out: output tensor
```

Comment:
What should the type of the output be? For example, if the inputs are quantized with scaling parameters, should the output also follow the same quantization, thus producing scaling parameters too?

I'm not sure what would be the best way, but it's probably just fine to make it whatever type most convenient to use with the Aten fallback.